SU-HSDO/suhumsci
# Arch 312 course filters

# READY FOR REVIEW

# Summary
- Added filters to course views.

# Needed By (Date)
- 8/1 (OMG August?!)

# Urgency
- low

# Steps to Test

1. checkout this branch
1. ensure to `composer update --prefer-source`
1. `blt drupal:sync --site=archaeology --partial`
1. `drush @archaeology.local mr hs_courses`
1. `drush @archaeology.local mim hs_courses`
1. go to the `/academics/courses` page and click on "Layout" on the top to change the layout.
1. in one of the sidebars add the "Exposed form: hs_courses-list" exposed filter
1. validate the filters work as expected.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
